src/Workspaces/Core/Portable/Serialization/SerializerService_Reference.cs

```diff
@@ -472,7 +472,7 @@ private static ModuleMetadata ReadModuleMetadataFrom(ObjectReader reader, Serial
         }
         else
         {
-            var service2 = (ITemporaryStorageService2)_storageService;
+            var service2 = (TemporaryStorageService)_storageService;
```

Comment:
this was the only impl of that interface.  so any blind casts to it are safe to move to a blind cast to the class itself.